plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -174,6 +179,11 @@ static obs_properties_t *ffmpeg_source_getproperties(void *data)
 				obs_module_text("InputFormat"),
 				OBS_TEXT_DEFAULT);
 
+	prop = obs_properties_add_int_slider(
+		props, "reconnect_delay_time",
+		obs_module_text("ReconnectDelayTime"), 2, 60, 1);
```

Comment:
```suggestion
		obs_module_text("ReconnectDelayTime"), 1, 60, 1);
```

I realized there's no reason not to make it 1s as the smallest amount.